Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "image"
-version = "0.21.0"
+version = "0.20.1"
```

Comment:
This is a breaking change so we can't use a minor version bump...

Reply:
This wasn't a minor version bump though, it was only changing it back to the version on master. Generally, this crate only bumps the version just before release. Also, the new version bump you added in https://github.com/PistonDevelopers/image/pull/819/commits/6a798c4235fff2618d58811a638822780a33d399 is invalid (it needs three numeric parts).